src/components/NFTPortfolio/Base.vue

```diff
@@ -1,78 +1,80 @@
 <template>
-  <NFTPortfolioCard>
-    <div
-      class="h-[180px]"
-      :style="`background-color: ${imageBgColor}`"
-    >
-      <img
-        v-if="imageSrc"
-        class="object-cover w-full h-full max-h-[180px]"
-        :src="imageSrc"
-      >
-      <img
-        v-else
-        class="object-cover w-full h-full max-h-[180px]"
-        src="~/assets/images/nft/primitive-nft.png"
-      >
-    </div>
-    <div
-      :class="[
-        'flex',
-        'flex-col',
-        'text-center',
-        'whitespace-pre-line',
-        'px-[24px]',
-        'pt-[48px]',
-        'py-[24px]',
-        'relative',
-      ]"
-    >
-      <div class="flex flex-col items-center justify-center mt-[-70px]">
-        <Identity
-          :avatar-url="userAvatarSrc"
-          :avatar-size="40"
-          :is-avatar-outlined="isUserCivicLiker"
-        />
-        <div class="flex mt-[8px]">
-          <Label class="text-medium-gray">by</Label>
-          <Label
-            class="text-like-green ml-[4px] font-[600]"
-          >{{ userDisplayName | ellipsis }}</Label>
-        </div>
-      </div>
-      <Label preset="h5" class="mt-[12px] break-words" align="center">{{ title }}</Label>
+  <NFTGemWrapper class="mb-[16px]" :collected-count="collectedCount">
```

Comment:
I think it is better to add spacing class, i.e. `mb-[16px]`, at the outermost component for reusability